slurp.1.scd

```diff
@@ -44,6 +44,12 @@ held, the selection is moved instead of being resized.
 	Set color for highlighting predefined rectangles from standard input when not
 	selected.
 
+*-F* _font family_
+	Set the font family name when displaying the dimensions box. Only useful
+	when combined with the -d option. The available font family names are the
+	standard generic CSS2 options: serif, sans-serif, monospace, cursive and
+	fantasy. Defaults to the Sans-Serif Family name.
```

Comment:
This makes it sound like "real" font names won't work, can we reword that?

Reply:
Well, if anything, the docs do mention that this API has no guarantee of working with "real" fonts.

> Selects a family and style of font from a simplified description as a family name, slant and weight. Cairo provides no operation to list available family names on the system (this is a "toy", remember), but the standard CSS2 generic family names [...] are likely to work as expected. [...] For "real" font selection, see the font-backend-specific `font_face_create` functions for the font backend you are using.

Do you still want me to reword anyway?

Reply:
Well, it just says that there's no way to get a list of the font names which will work, since that depends on the system's installed fonts. But "real" font names will definitely work fine. The CSS2 generic names just happen to be installed and configured on most systems.

IOW, if the user installs "Liberation", then configures slurp to use that font, it'll work fine.